cpp/src/arrow/dbi/hiveserver2/CMakeLists.txt

```diff
@@ -31,29 +31,32 @@ set(ARROW_HIVESERVER2_SRCS
     types.cc
     util.cc)
 
+set(HIVESERVER2_THRIFT_SRC_DIR "${ARROW_BINARY_DIR}/src/arrow/dbi/hiveserver2")
```

Comment:
I think our hiveserver adapter doesn't even build currently, do we want to bother fixing it?